src/components/Search/SearchAutocompleteList.tsx

```diff
@@ -290,6 +296,9 @@ function SearchAutocompleteList(
                     mapKey: CONST.SEARCH.SYNTAX_FILTER_KEYS.TAX_RATE,
                 }));
             }
+            case CONST.SEARCH.SYNTAX_FILTER_KEYS.CREATED_BY:
+            case CONST.SEARCH.SYNTAX_FILTER_KEYS.ASSIGNEE:
+            case CONST.SEARCH.SYNTAX_FILTER_KEYS.TO:
             case CONST.SEARCH.SYNTAX_FILTER_KEYS.FROM: {
```

Comment:
I think this might be an issue since in line 308 and 311 we're using `filterKey: CONST.SEARCH.SEARCH_USER_FRIENDLY_KEYS.FROM`. We also need to update that filter key dynamically